runtime_scan/pkg/orchestrator/scanconfigwatcher/helpers.go

```diff
@@ -31,6 +31,7 @@ func newScanFromScanConfig(scanConfig *models.ScanConfig) *models.Scan {
 			ScanFamiliesConfig:  scanConfig.ScanFamiliesConfig,
 			Scheduled:           scanConfig.Scheduled,
 			Scope:               scanConfig.Scope,
+			TimeoutSeconds:      scanConfig.TimeoutSeconds,
```

Comment:
maybe worth checking that this doesn't come negative from the api because you're doing calculations on it, or maybe doing it with min value in the API (not sure what it should be...)

Reply:
I'll check how we can add validation to this on the API side.

Reply:
@akpsgit I added validation to `isScanTimedOut` helper function so we only use `TimeoutSeconds` with value bigger than zero.